examples/optuna_search_cv_simple.py

```diff
@@ -0,0 +1,41 @@
+"""
+Optuna example that optimizes a classifier configuration using OptunaSearchCV.
+
+In this example, we optimize a classifier configuration for Iris dataset using OptunaSearchCV.
+Classifier is from scikit-learn.
+
+You can run this example as follow:
```

Comment:
Please replace "as follow" with "as follows".

Reply:
Thanks! I fixed it.